src/call/server.rs

```diff
@@ -119,7 +119,7 @@ impl RequestContext {
         self.ctx
     }
 
-    fn call(&mut self, cq: CompletionQueue) -> Call {
+    fn call(&self, cq: CompletionQueue) -> Call {
```

Comment:
It feels strange that `grpcwrap_request_call_context_ref_call` takes a mutable pointer while here use immutable borrow.